### PR TITLE
optionally mount og in vagrant

### PR DESCRIPTION
**Pull request by drunkirishcoder**

### CODE REVIEW

**Comment by zeeshanlakhani** [comment]
👍

File: Vagrantfile [hunk 1]
```diff
@@ -10,6 +10,7 @@ end
 $dimage = ENV.fetch("DIMAGE", "netbricks")
 $dtag = ENV.fetch("DTAG", "latest")
 $dproject = ENV.fetch("DPROJECT", "williamofockham")
+$ogpath = ENV.fetch("NBPATH", "../../og")
```
Comment (clinedome) [inline Vagrantfile]: should this be OGPATH ? Also, why two `../`?

Comment (zeeshanlakhani) [reply Vagrantfile]: TRUTH

Comment (drunkirishcoder) [reply Vagrantfile]: Yes, should be OGPATH. My bad.

I'm bringing in Utils, NetBricks, and MoonGen as submodules to the `og` project. I'm going up two levels and come back down to make sure it's not mounting the wrong folder as `og` if you are starting from `/tmp/utils` or something.

Comment (clinedome) [reply Vagrantfile]: Ok, word

Comment (drunkirishcoder) [reply Vagrantfile]: Fixed.

**Pull request merged**